blog/2017-08-17-debezium-0-5-2-is-out.adoc

```diff
@@ -0,0 +1,64 @@
+= Debezium 0.5.2 Is Out
+gmorling
+:awestruct-tags: [ releases, mysql, mongodb, postgres, docker ]
+:awestruct-layout: blog-post
+
+I'm very happy to announce the release of **Debezium 0.5.2**!
+
+As the previous release, 0.5.2 fixes several bugs in the link:/docs/connectors/mysql/[MySQL], link:/docs/connectors/postgresql/[Postgres] and link:/docs/connectors/mongodb/[MongoDB] connectors.
+But there are also several new features and options:
+
+* The `decimal.handling.mode` option already known from the MySQL connector is now also supported for PostgreSQL (https://issues.jboss.org/browse/DBZ-337[DBZ-337]).
+It lets you control how `NUMERIC` and `DECIMAL` columns are represented in change events (either using Kafka's `Decimal` type or as `double`).
+* The MongoDB connector supports the options `database.whitelist` and `database.blacklist` now (https://issues.jboss.org/browse/DBZ-302[DBZ-302])
+* The PostgreSQL connector can deal with array-typed columns as well as with quoted identifiers for tables, schemas etc. (https://issues.jboss.org/browse/DBZ-297[DBZ-297], https://issues.jboss.org/browse/DBZ-298[DBZ-298])
+* The Debezium Docker images run on Red Hat's https://www.openshift.com/[OpenShift] cloud environment (https://issues.jboss.org/browse/DBZ-267[DBZ-267])
+
+Speaking about the Docker images, we've set up _nightly_ tags for the https://hub.docker.com/u/debezium/[Debezium images on Docker Hub],
+allowing you to grab the latest improvements even before an official release has been cut.
+The connector archives are also deployed to the https://oss.sonatype.org/content/repositories/snapshots/io/debezium/[Sonatype OSS Maven repository].
+
+Finally, we've spent some time to extend the documentation on some things not covered before:
+
+* http://debezium.io/docs/configuration/avro/[Avro Serialization] describes how to use the Avro converter - together with the Confluent schema registry - instead of the default JSON converter for serializing change events, resulting in much smaller message sizes;
```

Comment:
"... use the Confluent Schema Registry and the Avro converter instead of the JSON converter ..."